starboard/nplb/file_helpers.h

```diff
@@ -43,7 +43,7 @@ std::vector<std::string> GetFileTestsDirectoryPaths();
 std::string GetTestFileExpectedContent(const std::string& path);
 
 // Removes directory and all of its contents.
```

Comment:
I'd recommend just removing this comment, it's already out of date and the function name explains it